docs/standard/analyzers/api-analyzer.md

```diff
@@ -34,16 +46,31 @@ The **Error List** window contains warnings with a unique ID per deprecated API,
 
 By clicking on the ID, you go to a webpage with detailed information about why the API was deprecated and suggestions regarding alternative APIs that can be used.
 
-Any warnings can be suppressed by right-clicking on the highlighted member and selecting **Quick Actions and Refactorings**. There are two ways to supress warnings: 
+Any warnings can be suppressed by right-clicking on the highlighted member and selecting **Suppress *diagnostic ID***. There are two ways to supress warnings: 
 
 * locally (in source)
 * globally (in a suppression file)
 
+### Suppresing warnings localy
+
+To suppress warnings localy you need to right-click on the member you want to suppress warnings for and select **Suppress *diagnostic ID***, then click **in Source**. Your code will be framed with `#pragma` like it is shown below:
```

Comment:
nit: missing comma after To suppress warnings localy

you need to right-click -> right-click

typo: localy -> locally

it seems you do need to click Quick Actions and Refactorings first, no?

You could put the diagnostic ID in brackets like **Suppress \<diagnostic ID>**

After selecting in Source, they also get options on the scope of the change. By default, it's only on that specific call. 

you should not use below. Replace with following. 

add link to #pragma warning docs? https://docs.microsoft.com/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-pragma-warning? (../../csharp/language-reference/preprocessor-directives/preprocessor-pragma-warning.md)

Try to avoid future tense. You could say something like: "The #pragma warning preprocessor directive is added to your source code in the scope defined."